sh_script/fuzzing.sh

```diff
@@ -135,9 +137,11 @@ run_single_case() {
     echo $test_case | grep "^afl"
     if [ "$?" == "0" ];then
         echo "The test method is afl"
-        export RUSTFLAGS="-C instrument-coverage"
-        export LLVM_PROFILE_FILE="fuzz-%p-%m.profraw"
-        find . -name "*.profraw" | xargs rm -rf
+        if [ "${collect_coverage}" == "YES" ]; then
+            export RUSTFLAGS="-C instrument-coverage"
+            export LLVM_PROFILE_FILE="fuzz-%p-%m.profraw"
+            find . -name "*.profraw" | xargs rm -rf
```

Comment:
This makes an assumption that the script is being run from the top-level repo dir. If it isn't, the user might get a surprise! It might be safer to determine the top-level directory based on the directory this script lives in to avoid any possibly surprises.